Mathlib/Data/Polynomial/Div.lean

```diff
@@ -555,8 +567,10 @@ set_option linter.uppercaseLean3 false in
 /- Porting note: stripping out classical for decidability instance parameter might
 make for better ergonomics -/
 /-- The largest power of `X - C a` which divides `p`.
-This is computable via the divisibility algorithm `Polynomial.decidableDvdMonic`. -/
+This *could be* computable via the divisibility algorithm `Polynomial.decidableDvdMonic`,
+as shown by `Polynomial.rootMultiplicity_eq_nat_find_of_nonzero` which has a computable RHS. -/
 def rootMultiplicity (a : R) (p : R[X]) : ℕ :=
+  letI := Classical.decEq R
```

Comment:
Making this computable is out of scope for this PR (and it would be a tradeoff that people might not want anyway).